uiowa/uiowa
# Superfish menu bugs

# How to test

Closes https://github.com/uiowa/uiowa/issues/3454
Closes #4276
Closes https://github.com/uiowa/uiowa/issues/4279


blt dsa sites ( and test menus at different breakpoints to make sure bugs are fixed. 
Example sites: 

- africanamericanstudies.uiowa.edu
- cinematicarts.uiowa.edu
- nursing.uiowa.edu

make sure updates are imported (drush @site-alias.local cim)
`blt frontend`
` drush @site-alias.local cr` `drush @cinematicarts.local cr` `drush @africanamericanstudies.local cr` `drush @nursing.local cr`

Make sure nested menu links aren't cut off at breakpoints - limited max width of units
Hoping to kill two birds with one stone. There's additionally a superfish menu bug where the drop down arrow overlaps the menu link/text in tablet and mobile views. Prod example: ![image](https://user-images.githubusercontent.com/16872949/136315187-14b3c42f-6d58-4716-a2d2-e1c31ca1e4b9.jpeg)
